.changeset/upset-ducks-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/unenv-preset": patch
+---
+
+Use native node:vm module when available
```

Comment:
Could you please add details about the default enable compat date and the compat flag